comprobo_final_project/scripts/april_tags.py

```diff
@@ -0,0 +1,53 @@
+#!usr/bin/env python
+
+"""
+Got Apriltags absolute position
+"""
+
+import rospy
+from geometry_msgs.msg import Twist, Point, PoseStamped
+import tf
+
+class April(object):
+    """
+    Basically for now it just takes data from the Apriltags and can send it somewhere else
+    """
+
+    def __init__(self):
```

Comment:
Check out the latest GazeboPositionProvider, one thing we can do is pass in `rospy` here, and let another class, say `RobotController` take care of the rest